### PR TITLE
Omit AppleDouble encoded Macintosh files. Useless in other Unix-like …

### PR DESCRIPTION
**Pull request by menivaitsi**
…systems.

https://ocd.appscale.com:8080/job/Daily%20Build/807/

### CODE REVIEW

File: lib/remote_helper.py [hunk 1]
```diff
@@ -989,7 +989,7 @@ def copy_app_to_host(cls, app_location, keyname, is_verbose):
     rand = str(uuid.uuid4()).replace('-', '')[:8]
     local_tarred_app = "{0}/appscale-app-{1}-{2}.tar.gz".format(tempfile.gettempdir(),
       app_id, rand)
-    LocalState.shell("cd '{0}' && tar -czhf {1} --exclude='*.pyc' *".format(
+    LocalState.shell("cd '{0}' && COPYFILE_DISABLE=1 tar -czhf {1} --exclude='*.pyc' *".format(
```
Comment (obino) [inline lib/remote_helper.py]: this will disable all . file, correct? If so, it may be too draconian.

Comment (menivaitsi) [inline lib/remote_helper.py]: Funny you should mention Draco; my favorite legislator. 
Joking aside, it will only omit Apple metadata files used for HFS+.

Comment (obino) [inline lib/remote_helper.py]: very draconian. I like it. :+1:

**Comment by menivaitsi** [comment]
OCD: https://ocd.appscale.com:8080/job/Daily%20Build/812/

**Pull request merged**